docs/participate/translate_space_acres.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Space Acres UI Translation Guide
+sidebar_position: 4
+description: Translation Guide for the Space Acres UI
+keywords:
+    - Space Acres UI translation
+    - Community
+    - Autonomys
+    - Translate
+    - Localization
+---
+
+
+## Translation Guide
+
+Thank you for considering contributing to the Space Acres UI locallization! This guide is here to help you contribute to our goal of making farming on the Autonomys Network more accessible and inclusive by localizing or translating the user interface. The Autonomys Network is driven by a vision of a decentralized and equitable future, and we believe that overcoming language barriers is crucial to achieving this vision.
+
+## Before you start
+
+Space Acres has a lot of techncical and Autonomys-related slang that **will not be translated correctly** via the AI-chats similar to **Chat GPT** or **Google Translate**. We kindly ask you to work to do all translations manually, since we can't really verify their correctness. We thank you for your understanding! 
+
+## Step by Step Tutorial
+
+1. Sign in into your [GitHub](https://github.com/) account, or create a new account if you don't have one yet. 
+2. Proceed to [Space Acres translations folder](https://github.com/autonomys/space-acres/tree/main/res/translations) in the Space Acres reposotory.
+
+    ![sa-translate-1](/img/doc-imgs/space-acres-translate/sa-translate-1.png)
+
+3. Check that the language you're planning to translate the UI to **has not been translated yet**! 
+4. Click on **Add File** and select **Create New File**.
+    
+    ![sa-translate-2](/img/doc-imgs/space-acres-translate/sa-translate-2.png)
+
+    
+5. Provide the name of the file: it should start with the two letters from [an IBAN Alpha-2 code](https://www.iban.com/country-codes) list followed by the `/messages.ftl`. 
+
+    ![sa-translate-3](/img/doc-imgs/space-acres-translate/sa-translate-3.png)
+
+    For example, if you're willing to translate the Space Acres UI into **Russian language**, you would put `ru/messages/ftl`. You will notice that as soon as you put the forward slash `/` after the double-digit language code, it would turn **blue**, meaning that your translation file will be created in a folder, which is correct and expected!
```

Comment:
```suggestion
    For example, if you're planning to translate the Space Acres UI into the **Russian language**, you would put `ru/messages/ftl`. You will notice that as soon as you put the forward slash `/` after the 2-character language code, it will turn **blue**, meaning that your translation file will be created in a folder which is correct and expected.
```